R/data_mile2.R

```diff
@@ -0,0 +1,38 @@
+#' FIMS input data frame for milestone 2
+#'
+#' A dataset containing information necessary to run an age-structured stock
+#' assessment model with length composition data in FIMS for milestone 2 . 
+#' This data was generated using
+#' the `ASSAMC` package written for the [model comparison project](
+#' www.github.com/Bai-Li-NOAA/Age_Structured_Stock_Assessment_Model_Comparison).
+#'
+#' @format A data frame with `r NROW(data_mile2)` observations of
+#' `r NCOL(data_mile2)` variables:
+#' \describe{
+#'   \item{type}{The type of data the row contains. Allowed types include
+#'     `age`, `length-comp`, `length-at-age`, `index`, `landings`, and `weight-at-age` data.}
+#'   \item{name}{A character string providing the name of the information source
+#'     that the data was collected from, e.g., `"Trawl fishery"`}
+#'   \item{age}{An integer age. Entry can be `NA` if information pertains to
+#'     multiple ages, e.g., total catch rather than catch of age-4 fish.}
+#'   \item{len}{An integer length. Entry can be `NA` if information pertains to
```

Comment:
Please use "length" rather than "len". We are going for self-documenting code rather than abbreviations. I am assuming that some other code will need to be changed as well to accommodate this.

Reply:
The reasoning for this was to avoid using length specifically as it is a common R function which I thought we had previously discussed should not be used?

Reply:
Changed this to length